astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,71 @@
+---
+publish_date: 2023-10-19
+title: I Just Want Authentication To Work
+description: When I am writing an Application the last thing that I care about is setting up Authentication, I just want it to work!
+authors: Alex Patterson
+image: /img/blogs/i-just-want-authentication-to-work/i-just-want-auth-to-work.png
+categories: Company, News
+tags: award, security mag
+excerpt_separator: "{/* more */}"
+---
+
+Authentication is the process of verifying the identity of a user or entity. It is a fundamental security measure that is used in a wide variety of applications, including web applications, mobile apps, and APIs. The thing is I just don't care about figuring it out! I just want it to work!!
```

Comment:
Who is "I" here? Knowing you are an experienced dev lets me know if I should value your opinion or not.